packages/barcode-scanning/ios/Plugin/BarcodeScannerHelper.swift

```diff
@@ -81,10 +81,37 @@ public class BarcodeScannerHelper {
             result["bytes"] = convertDataToJsonArray(rawData)
         }
         result["cornerPoints"] = cornerPointsResult
+        if let barcode.calendarEvent {
+            result["calendarEvent"] = extractCalendarEventProperties(barcode.calendarEvent!)
+        }
```

Comment:
```suggestion
        if let calendarEvent = barcode.calendarEvent {
            result["calendarEvent"] = extractCalendarEventProperties(calendarEvent)
        }
```

Sorry, I think I had a syntax error in my last suggestion. At the moment you are still doing a force unwrap which is bad practice. This is how it should work.

Reply:
sorry it was my carelessness, hope it's finally fine now 😅